doc/api/errors.md

```diff
@@ -703,6 +703,13 @@ in some cases may accept `func(undefined)` but not `func()`). In most native
 Node.js APIs, `func(undefined)` and `func()` are treated identically, and the
 [`ERR_INVALID_ARG_TYPE`][] error code may be used instead.
 
+<a id="ERR_NO_CRYPTO"></a>
+### ERR_NO_CRYPTO
+
+An error using the `'ERR_NO_CRYPTO'` error code is thrown specifically when 
```

Comment:
Please adapt this to match the wording used in #13627 ("Used when an attempt is made...").